doc/intro.rst

```diff
@@ -63,3 +61,93 @@ that sets its character:
   packages for details. Not enough? If you
   need a full, standalone GUI, `HyperSpyUI <https://hyperspy.org/hyperspyUI/>`_
   is for you.
+
+Learning resources
+==================
+
+.. grid:: 2 3 3 3
+  :gutter: 2
+
+  .. grid-item-card::
+    :link: user_guide/index.rst
```

Comment:
```suggestion
    :link: user_guide/index.html
```
At least on the `Read the Docs build`, linking to the `.rst` does not work. It gives a `404` error.

Reply:
Currently the "Getting started" and the "User guide" card links to the same page: `user_guide/index.html`. Should:
- `Getting started` link to `user_guide/install.html`?
- `User guide` link to `user_guide/index.html`?